imagetest/local_build.sh

```diff
@@ -0,0 +1,11 @@
+# rootpath should be an absolute path where you want test output files storedm, such as /home/username
+# suite is the name of the test suite
+rootpath=$1
+suite=$2
+go build -o $rootpath/wrapper.amd64 ./cmd/wrapper/main.go
```

Comment:
Added an option -i $imagetestroot, which can be passed in.

If the script is invoked from guest-test-infra, the caller could add -i imagetest (./imagetest/local_build.sh -o $outspath -s disk -i imagetest) so that the script knows the path to imagetest.